sdk/keyvault/keyvault-keys/src/localCryptography/conversions.ts

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { JsonWebKey } from "../keysModels";
+
+/**
+ * @internal
+ * @ignore
+ * Encodes a length of a packet in DER format
+ */
+function encodeLength(length: number): Uint8Array {
+  if (length <= 127) {
+    return Uint8Array.of(length);
+  } else if (length < 256) {
+    return Uint8Array.of(0x81, length);
+  } else if (length < 65536) {
+    return Uint8Array.of(0x82, length >> 8, length & 0xff);
+  } else {
+    throw new Error("Unsupported length to encode");
+  }
+}
+
+/**
+ * @internal
+ * @ignore
+ * Encodes a buffer for DER, as sets the id to the given id
+ */
+function encodeBuffer(buffer: Uint8Array, bufferId: number): Uint8Array {
+  if (buffer.length === 0) {
+    return buffer;
+  }
+
+  let result = new Uint8Array(buffer);
+
+  // If the high bit is set, prepend a 0
+  if (result[0] & 0x80) {
```

Comment:
This was previously `result[0] & 0x80 === 0x80`, but these are the same I believe. Am I wrong?